pkg/kubelet/certificate/bootstrap/bootstrap_test.go

```diff
@@ -34,9 +35,263 @@ import (
 	certificatesclient "k8s.io/client-go/kubernetes/typed/certificates/v1beta1"
 	restclient "k8s.io/client-go/rest"
 	clienttesting "k8s.io/client-go/testing"
+	"k8s.io/client-go/util/certificate"
 	"k8s.io/client-go/util/keyutil"
 )
 
+func copyFile(src, dst string) (err error) {
+	in, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer in.Close()
+	out, err := os.Create(dst)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		cerr := out.Close()
+		if err == nil {
+			err = cerr
+		}
+	}()
+	_, err = io.Copy(out, in)
+	return err
+}
+
+func TestLoadClientConfig(t *testing.T) {
+	//Create a temporary folder under tmp to store the required certificate files and configuration files.
+	fileDir, err := ioutil.TempDir("", "dir-")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(fileDir)
```

Comment:
use t.TempDir instead? (https://golang.org/pkg/testing/#T.TempDir)

Reply:
Got it! Learned it!